openwisp_monitoring/monitoring/static/monitoring/js/chart.js

```diff
@@ -193,6 +193,40 @@
             charts.push(options);
         }
         charts = sortByTraceOrder(data.trace_order, charts, '_key');
+        if(data.trace_order !== undefined)  {
+            var total_traffic_charts = charts[0].y;
+            var sum = 0;
+            for(i=0; i<total_traffic_charts.length; i++){
+                sum += total_traffic_charts[i];
+            }
```

Comment:
Nitpick, you don't *need* to do this change, but just want to teach you a cleaner way to do this:

```
const sum = total_traffic_charts.reduce((sum, chart) => sum = sum + chart, 0);
```

Why is this better? Pl. google `pure functions`. :smile:

Reply:
I agree this looks cleaner :+1: